clients/python/README.md

```diff
@@ -0,0 +1,69 @@
+# Stencil Python client
+
+[![PyPI version](https://badge.fury.io/py/stencil-python-client.svg)](https://pypi.org/project/stencil-python-client)
+
+Stencil Python client package provides a store to lookup protobuf descriptors and options to keep the protobuf descriptors upto date.
+
+It has following features
+ - Deserialize protobuf messages directly by specifying protobuf message name
+ - Ability to refresh protobuf descriptors in specified intervals
+ - Support to download descriptors from multiple urls
+
+
+## Requirements
+
+ - Python 3.7+
+
+## Installation
+
+Use `pip`
+```
+pip3 install stencil-python-client
```

Comment:
Should we update this to use the git based method for downloading ?

Reply:
added